CONTRIBUTING.md

```diff
@@ -7,7 +7,7 @@ If you just want to contribute a client library in a new language and not make c
 ## Requirements
 
 - MacOS or Linux (Windows may work too, but we haven't tested it)
-- [NodeJS](https://nodejs.org/en/download/package-manager/) 16.x or higher
+- [NodeJS](https://nodejs.org/en/download/package-manager/) 18.x
```

Comment:
because of microsoft azure we are unable to go above version 18 so i updated copy to state that it is version 18 only